src/Database/Esqueleto/Internal/Internal.hs

```diff
@@ -461,6 +461,11 @@ not_ (ECompositeKey _) = throw (CompositeKeyErr NotError)
 (*.)  :: PersistField a => SqlExpr (Value a) -> SqlExpr (Value a) -> SqlExpr (Value a)
 (*.)  = unsafeSqlBinOp " * "
 
+-- | @BETWEEN@.
+--
+-- /Since: 3.0.0/
```

Comment:
This isn't going to be right, since `3.0.0` is already released. It'll batch up with 3.1.0.

```suggestion
-- @since 3.1.0
```